rareapi/views/Subscriptions.py

```diff
@@ -36,3 +37,64 @@ def create(self, request):
             return Response({}, status=status.HTTP_201_CREATED)
         except Exception as ex:
             return Response({'message': ex.args[0]}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+    """action to update the ended_on date for when someone unsubscribes"""
+    @action(methods=['put'], detail=False)
+    def update_subscription(self,request):
```

Comment:
Could we name this custom action "end_subscription"? I think that would clarify what we are trying to achieve